guacamole/src/main/webapp/translations/ca.json

```diff
@@ -0,0 +1,962 @@
+{
+    
+    "NAME" : "Catalan",
+    
+    "APP" : {
+
+        "NAME"    : "Apache Guacamole",
+        "VERSION" : "${project.version}",
```

Comment:
Both of these lines should be left out of this translation file.

Reply:
Ok. I Do it.

Sorry. I'm newie with git & github. I must re-do the pull request when I correct all ?

Thank you

Reply:
No, you do not need to close and redo the pull request - just make the changes and push your commits to your working branch, and the changes will show up, here.  You can also rebase to correct the commit messages and then force-push with the new commit messages.

Reply:
APP.NAME & APP.VERSION removed from translation